stable/ambassador/README.md

```diff
@@ -196,6 +196,12 @@ $ helm upgrade --install --wait my-release -f values.yaml stable/ambassador
 
 # Upgrading
 
+## To 5.0.0
+
+### Breaking changes
+
+- Helm 3 support for CRDs was added. Specifically, the CRD templates were moved to non-templated files in the `/crds` directory, and to keep Helm 2 support they are globbed from there by `/templates/crds.yaml`. However, because Helm 3 CRDs are not templated, the labels have necessarily changed. Because of this, you must manually delete any ambassador CRDs installed by previous versions of this chart before upgrading
```

Comment:
```suggestion
- Helm 3 support for CRDs was added. Specifically, the CRD templates were moved to non-templated files in the `/crds` directory, and to keep Helm 2 support they are globbed from there by `/templates/crds.yaml`. However, because Helm 3 CRDs are not templated, the labels have necessarily changed. Because of this, you must manually delete any ambassador CRDs installed by previous versions of this chart before upgrading

**Note** If upgrading a helm 2 based install no action is needed, previously installed CRDs will not be modified.
```

Reply:
Same as above